tedana/reporting/data/html/report_body_template.html

```diff
@@ -286,6 +286,17 @@ <h3>Global Signal Timeseries</h3>
     </div>
 </div>
 {% endif %}
+{% if externalRegressorsExist %}
+<div class="carpet-plots">
```

Comment:
```suggestion
<div class="carpet-plots">
  <div class="carpet-plots-image">
```

Reply:
This should make the image as big as the carpet plots. It think it would be nice to make them the same size for consistency.

Reply:
That's awesome, thank you!